src/clusterfuzz/_internal/bot/fuzzers/libfuzzer.py

```diff
@@ -932,12 +932,32 @@ def _copy_local_directories_from_device(self, local_directories):
       android.adb.copy_remote_directory_to_local(device_directory,
                                                  local_directory)
 
-  def _append_logcat_output_if_needed(self, output):
+  def _prepend_trusty_stacktrace_if_needed(self, output):
+    """Add trusty stacktrace to beginning of output if found in logcat."""
+    logcat = android.logger.log_output()
+    begin, end = '---------', 'Built:'
+    target = 'Backtrace for thread:'
```

Comment:
It was asked if these markers would be stable. `Backtrace for thread` and `Built:` are messages added in the Trusty code for dumping their backtraces, so I believe these markers will be stable. `---------` appears to be a standard marker for indicating the beginning of an individual kernel.log entry.